documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1908,6 +1908,18 @@ Emitting the tombstone event (the default behavior) allows Kafka to completely d
 |_n/a_
 |An optional comma-separated list of regular expressions that match the fully-qualified names of character-based columns whose values should be replaced in the change event message values with a field value consisting of the specified number of asterisk (`*`) characters. Multiple properties with different lengths can be used in a single configuration, although in each the length must be a positive integer or zero. Fully-qualified names for columns are of the form _databaseName_._tableName_._columnName_, or _databaseName_._schemaName_._tableName_._columnName_.
 
+|`column.mask.hash._hashAlgorithm_.with.salt._salt_`
+|_n/a_
+|An optional comma-separated list of regular expressions that match the fully-qualified names of character-based columns whose values should be pseudonyms in the change event message values with a field value consisting of the hashed value using the algorithm `_hashAlgorithm_` and salt `_salt_`.
+Based on the used hash function referential integrity is kept while data is pseudonymized. Supported hash functions are described in the {link-java7-standard-names}[MessageDigest section] of the Java Cryptography Architecture Standard Algorithm Name Documentation.
+Multiple properties with different lengths can be used in a single configuration, although in each the length must be a positive integer or zero. Fully-qualified names for columns are of the form _databaseName_._tableName_._columnName_, or _databaseName_._schemaName_._tableName_._columnName_.
+
+Example:
+
+    column.mask.hash.SHA-256.with.salt.randomSalt = databaseName.tableOrder.columnCustomerName, databaseName.tableShipment.columnCustomerName
```

Comment:
Could we make it a truly complete? In particular, how is the salt to be given? I don't think it can be a random value, as otherwise the same string would be hashed into different values after a connector restart, defeating the purpose of the feature. I.e. the salt value must be given here explicitly.

Reply:
randomSalt should be randomly selected by the user. As long as the reference integrity should be preserved, randomSalt should remain unchanged.

I should better emphasize that randomSalt has to be selected by the user and has to be entered explicitly in the configuration. I'm going to clarify it.

Reply:
Yes, let's make it really explict. also `columnCustomerName` should be `customerName` etc.